api.planx.uk/modules/webhooks/service/lowcalSessionEvents/schema.ts

```diff
@@ -4,7 +4,7 @@ import { ScheduledEventResponse } from "../../../../hasura/metadata";
 
 export const createSessionEventSchema = z.object({
   body: z.object({
-    createdAt: z.string().transform((val) => new Date(val)),
+    createdAt: z.string().pipe(z.coerce.date()),
```

Comment:
This is functionally the same, but syntactically a little nicer https://github.com/colinhacks/zod/discussions/879#discussioncomment-4646183